### ISSUE TITLE
codesign issue in macos

### ISSUE THREAD

**Issue by yihuang** [3noch/nix-bundle-exe#8]
I have this issue on macos 12.3.1:
```
dyld[76184]: Library not loaded: @executable_path/../Frameworks/Library.dylib/librocksdb.6.26.0.dylib
  Referenced from: /private/tmp/cronosd/bin/cronosd
  Reason: tried: '/private/tmp/cronosd/bin/../Frameworks/Library.dylib/librocksdb.6.26.0.dylib' (code signature in <no uuid> '/private/tmp/cronosd/Frameworks/Library.dylib/librocksdb.6.26.0.dylib' not valid for use in process: library load disallowed by system policy), '/usr/local/lib/librocksdb.6.26.0.dylib' (no such file), '/usr/lib/librocksdb.6.26.0.dylib' (no such file)
```

The above error happens if I try to run a binary downloaded from web, but if I build it locally, it can run, even if the hash of binaries are identical.

**Comment by yihuang** [comment]
One way to fix it is by remove the quarantine attribute on the downloaded file:
```
xattr -d com.apple.quarantine binary.tar.gz
```

**Comment by yihuang** [comment]
I think it's because we modified the files after codesign: https://github.com/3noch/nix-bundle-exe/blob/main/default.nix#L36

**Comment by yihuang** [comment]
> Is this fixed by #9 ?

yes, closing now.
